docs/iris/src/whatsnew/aggregate_directory.py

```diff
@@ -238,7 +238,10 @@ def compile_directory(directory, release):
         for file_description in category_items:
             entry_path = os.path.join(directory, file_description['FileName'])
             with open(entry_path, 'r') as content_object:
-                category_text.extend(content_object.readlines())
+                text = content_object.readlines()
+                if not text[-1].endswith('\n'):
+                    text[-1] += '\n'
+                category_text.extend(text)
```

Comment:
Otherwise, for this release we'd end up with the following ...
```
Iris 1.11 Features
==================
* If available, display the ``STASH`` code instead of ``unknown / (unknown)`` when printing cubes
  with no ``standard_name`` and no ``units``.
* Support for saving to netCDF with data packing has been added.* The coordinate system :class:`iris.coord_systems.LambertAzimuthalEqualArea` has been added with NetCDF saving support.
```